Dockerfile.src

```diff
@@ -6,12 +6,14 @@ FROM openshift/origin-cli:latest as cli
 
 FROM openshift/origin-release:golang-1.13
 
-# our copy of faq and jq
-#COPY hack/faq.repo /etc/yum.repos.d/ecnahc515-faq-epel-7.repo
+# go get faq via static Linux binary approach
+ARG LATEST_RELEASE=0.0.6
+RUN curl -Lo /usr/local/bin/faq https://github.com/jzelinskie/faq/releases/download/$LATEST_RELEASE/faq-linux-amd64
+RUN chmod +x /usr/local/bin/faq
```

Comment:
I think another approach we could take is grabbing that binary from jimmy's quay account. Not sure if that's any better though.

Reply:
https://quay.io/repository/jzelinskie/faq?tag=latest&tab=tags

Reply:
Yep, but likely ART build will require some other fix which we'll want to match here, so... punting for now.